process_environment.py

```diff
@@ -471,7 +471,16 @@ def extract_light(self):
         print output_log
 
     def extract_lights(self):
-        cmd = "{} {}".format(extractLights_cmd, self.panorama_highres)
+
+        # compute it one time for panorama
+        img_size_x = 1024
```

Comment:
half resolution as in 512x1024 ?
yes, tried but on our bigger area light, it gave different light direction
( as the smaller the image, the biggest the "cone" of the light direction becomes. )